install/scripts/start-bn.sh

```diff
@@ -138,6 +138,15 @@ if [ "$CC_CLIENT" = "lodestar" ]; then
         CMD="$CMD --checkpointSyncUrl $CHECKPOINT_SYNC_URL"
     fi
 
+    if [ "$ENABLE_BITFLY_NODE_METRICS" = "true" ]; then
+        CMD="$CMD --monitoring.endpoint $BITFLY_NODE_METRICS_ENDPOINT?apikey=$BITFLY_NODE_METRICS_SECRET&machine=$BITFLY_NODE_METRICS_MACHINE_NAME"
+
+        # monitoring relies on metrics data
+        if [ "$ENABLE_METRICS" != "true" ]; then
```

Comment:
As monitoring relies on metrics data, it is required that metrics are also enabled by supplying the `--metrics` flag. This is no longer required on https://github.com/ChainSafe/lodestar/pull/5328 is merged an released, so might just wait for this before merging changes in this PR.

Reply:
Ok, I'll merge it once the next Lodestar release is out.

Reply:
Perfect, I just realized that the UX could really be improved when creating this PR putting, marking as draft until then